### PR TITLE
Please review card and input components. 

### PR DESCRIPTION
**Pull request by muskanbagrecha**

### CODE REVIEW

**Comment by muskanbagrecha** [comment]
Please review card and input component!

File: card/card.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
+.card{
+    display: flex;
+    position: relative;
+    font-family: 'Source Serif 4', sans-serif;
+    color: var(--color-complementary);
+    background-color: #272626;
+    background: rgba(255, 255, 255, 0.2);
+    margin: var(--spacing-medium);
+    border-radius: var(--bradius-small);
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
```
Comment (rahulyadav139) [inline card/card.css]: everything is great but you can declare colours at one central file.

**Review by rahulyadav139 — APPROVED** [review]

File: card/card.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+} 
+
+.card__content *{
+    margin: 2px;
+}
+
+.card-horizontal{
+    max-width: 70rem;
+    min-width: min-content;
+    max-height: fit-content;
```
Comment (anshulraheja) [inline card/card.css]: What I think, there is no need for min-width and max-height 
1. It has no effect. The content will itself will take the minimum required width 
2. you are using media query so it no it turns into a vertical card and hence this code never gets applied.
3. Even max-height has no effect as the content will itself take minimum space required.

File: input/input.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+.input-form>*{
+    display: block;
+    margin: 0.8rem 0;
+    font-family:'Source Serif 4', sans-serif;
+    font-size: var(--fsize-200);
+}
+
+.input-form label{
+    display: block;
```
Comment (anshulraheja) [inline input/input.css:15]: you already have given block property to all elements on line number 7

**Review by anshulraheja — APPROVED** [review]
UI is clean!

**Review by PuneetP16 — APPROVED** [review]
Good to merge

**Review by saurabhchirde — COMMENTED** [review]
Looks good to merge.

**Pull request merged**